.scuba.yml

```diff
@@ -1,16 +1,57 @@
-image: docker.pkg.github.com/panix-os/panixbuild/panixbuild:3.1.1
+image: docker.pkg.github.com/panix-os/panixbuild/panixbuild:3.2.0
 
 aliases:
     # Building
-    build: make -j$(nproc)
+    build: make -j$(nproc) all
+    debug:
+      environment:
+        MODE: debug
+      script: make -j$(nproc)
```

Comment:
Any reason we can't do the `make MODE=debug` thing here? I feel like it would make the scuba file just a *tad* more terse.

Reply:
I could, but this is technically the correct way of doing it according to the scuba docs. Plus, as time goes on, we may add an ARCH variable.

Plus there's no real reason for people to look at this file often.